model-utils/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: content
+title: "Model Utils - RendrJS"
+---
+
+# Model Utils
+
+This class gives utility functions for models and collections.  Some functionality includes; an `instanceof` wrappers, factory methods, and getters for constructors based on the name.  The ModelUtils class is initialized in the [app initialization](/app#constructor) process, and is attached to the [app](/app) object.
```

Comment:
This class **provides** utility functions for models and collections, **including:**  an `instanceof` wrappers...